src/FSharp.AWS.DynamoDB/Expression/ProjectionExpr.fs

```diff
@@ -28,7 +28,9 @@ type AttributeId with
             if List.isEmpty rest then
                 false
             else
-                sprintf "Document path '%s' not found." id.Name |> KeyNotFoundException |> raise
+                sprintf "Document path '%s' not found." id.Name
+                |> KeyNotFoundException
+                |> raise
```

Comment:
argh! I guess the rule is >=2 pipes means multi line
I'd prob write `raise (KeyNotFoundException($"..."))` if it did this to me
but fine